### PR TITLE
fix edgeless edit bar tooltip

### PR DESCRIPTION
**Pull request by m1911star**
related issue: https://github.com/toeverything/AFFiNE/issues/199

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ For help, discussion about best practices, or any other conversation that would
   <tr>
     <td align="center"><a href="https://github.com/westongraham"><img src="https://avatars.githubusercontent.com/u/89493023?v=4?s=50" width="50px;" alt=""/><br /><sub><b>Weston Graham</b></sub></a><br /><a href="https://github.com/toeverything/AFFiNE/commits?author=westongraham" title="Documentation">📖</a></td>
     <td align="center"><a href="https://github.com/pointmax"><img src="https://avatars.githubusercontent.com/u/49361135?v=4?s=50" width="50px;" alt=""/><br /><sub><b>pointmax</b></sub></a><br /><a href="https://github.com/toeverything/AFFiNE/commits?author=pointmax" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/m1911star"><img src="https://avatars.githubusercontent.com/u/4948120?v=4?s=50" width="50px;" alt=""/><br /><sub><b>m1911star</b></sub></a><br /><a href="https://github.com/toeverything/AFFiNE/commits?author=m1911star" title="Code">💻</a></td>
```
Comment (DiamondThree) [inline README.md:221]: maybe, you can delete this modification, because our “MD” is automatically generated, thanks

Comment (m1911star) [reply README.md:221]: ok

**Review by DiamondThree — APPROVED** [review]

File: libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx [hunk 2]
```diff
@@ -52,18 +52,18 @@ const tools: Array<{
     { type: 'frame', label: 'Frame', tooltip: 'Frame', icon: FrameIcon },
     {
         type: TDShapeType.Editor,
-        label: 'Text',
-        tooltip: 'Text',
+        label: 'TextBlock',
```
Comment (DiamondThree) [inline libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx:55]: I think using "Text Block" is better

File: libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx [hunk 3]
```diff
@@ -52,18 +52,18 @@ const tools: Array<{
...
     { type: 'draw', component: PenTools },
     { type: 'Connector', component: LineTools },
     // { type: 'erase', label: 'Erase', tooltip: 'Erase', icon: EraseIcon },
     {
-        type: TDShapeType.HandDraw,
-        label: 'HandDraw',
-        tooltip: 'HandDraw',
+        type: TDShapeType.HandDrag,
+        label: 'HandDrag',
+        tooltip: 'HandDrag',
```
Comment (DiamondThree) [inline libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx:66]: “Hand Drag”

Comment (m1911star) [reply libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx]: done

File: libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx (hunk 2 above)
Comment (m1911star) [reply libs/components/board-draw/src/components/tools-panel/ToolsPanel.tsx]: done

**Comment by lawvs** [comment]
Hey @m1911star ,thanks for your pull request! We have updated the contributors in https://github.com/toeverything/AFFiNE/pull/300!

**Pull request merged**